routines/davix.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import shutil
+import subprocess
+from diracos import diracoslib
+import os
+import glob
+import tarfile
+import tempfile
+from yum import rpmUtils
+
+# TODO: factorize with glite-ce-cream-cli.py, since it is essentialy the same thing
+
+CMAKE3_SLC6_INSTALLER = 'https://github.com/Kitware/CMake/releases/download/v3.19.3/cmake-3.19.3-Linux-x86_64.sh'
```

Comment:
```suggestion
CMAKE3_SLC6_INSTALLER = 'https://diracos.web.cern.ch/diracos/bootstrap/cmake-3.19.3-Linux-x86_64.sh'
```
Just to be safe have in "in house"

Reply:
good point !